db/migrate/20160816141116_remove_path_reservation_for_find_local_council.rb

```diff
@@ -6,6 +6,6 @@ def change
     # even completing the creation of the associated edition in publisher).
     # We want to reuse the route without leaving something lying around
     # that could accidently be used to alter that route from panopticon.
-    PathReservation.find_by(base_path: '/find-local-council').destroy
+    PathReservation.where(base_path: '/find-local-council').destroy_all
```

Comment:
Do we also want to check the owning app for this, so that we don't accidentally delete the new route somehow in the future?